content/flux/v0.x/release-notes.md

```diff
@@ -11,6 +11,19 @@ aliases:
 ---
 
 
+## v0.174.1 [2022-07-12]
+
+### Bug fixes
+
+- Update [`aggregate.mean()`](/flux/v0.x/stdlib/contrib/jsternberg/aggregate/mean/) 
+  and [`aggregate.sum()`](/flux/v0.x/stdlib/contrib/jsternberg/aggregate/sum/) 
+  to handle null values.
```

Comment:
This isn't referring to those two functions. This is a special optimization built into `aggregateWindow()` specifically when using `sum` and `mean` to aggregate data. This optimization wasn't handling null values correctly.

```suggestion
- Update [`aggregateWindow()`](/flux/v0.x/stdlib/universe/aggregatewindow/) 
  to correctly handle null values when using `sum` or `mean`.
```